arm-recoveryservices/2015-01-01/swagger/recoveryservices.json

```diff
@@ -0,0 +1,285 @@
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceOutput"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ResourceOutput"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "delete": {
+        "tags": [
+          "Vaults"
+        ],
+        "description": "Deletes a vault.",
+        "operationId": "Vaults_Delete",
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "name": "isGlobalRPContext",
+            "in": "query",
+            "required": false,
+            "type": "boolean"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "NoContent"
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "Object": {
+      "description": "Base of all objects.",
+      "type": "object",
+      "properties": {}
+    },
+    "ResourceCollectionOutput": {
+      "description": "The response model for Vault.",
+      "type": "object",
+      "properties": {
+        "Value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceOutput"
+          }
+        }
+      }
+    },
+    "ResourceOutput": {
+      "description": "Resource information, as returned by the resource provider.",
+      "type": "object",
+      "properties": {
+        "location": {
+          "description": "Location of the resource.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of the resource.",
+          "type": "string"
+        },
+        "etag": {
+          "description": "ETag of the Vault.",
+          "type": "string"
+        },
+        "tags": {
+          "description": "The tags attached to the resource.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "$ref": "#/definitions/ResourceProperties"
+        },
+        "id": {
+          "description": "The Id of the resource.",
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of the resource.",
+          "type": "string"
+        }
+      }
+    },
+    "Sku": {
+      "description": "Identifies the unique system identifier for each Azure resource.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Name.",
+          "type": "string"
+        }
+      }
+    },
+    "ResourceInput": {
```

Comment:
The ResourceInput and ResourceOutput have been modelled in a different way. Changing this would break the behavior for the clients and also would require a bit of remodelling in our service. So, we can't really take this suggestion at this point.